src/190B20.c

```diff
@@ -1856,17 +2643,86 @@ INCLUDE_ASM(s32, "190B20", func_80266978);
 
 void func_80266ADC(Actor* target) {
     target->unk_206 = -1;
-    target->flags |= 0x80000;
+    target->flags |= ACTOR_FLAG_80000;
 }
 
 void func_80266AF8(Actor* target) {
     target->unk_206 = 0;
-    target->flags &= ~0x80000;
+    target->flags &= ~ACTOR_FLAG_80000;
 }
 
-INCLUDE_ASM(s32, "190B20", func_80266B14);
+void func_80266B14(void) {
+    s32 i;
 
+    for (i = 0; i < ARRAY_COUNT(gBattleStatus.enemyActors); i++) {
+        Actor* enemy = gBattleStatus.enemyActors[i];
+
+        if (enemy != NULL) {
+            if (enemy->unk_206 > 0) {
+                enemy->unk_206--;
+                if (enemy->unk_206 == 0) {
+                    enemy->flags &= ~ENEMY_FLAGS_80000;
+                }
+            }
+        }
+    }
+}
+
+// dumb label
+#ifdef NON_MATCHING
+s32 try_inflict_status(Actor* actor, s32 statusTypeKey, s32 statusKey) {
+    BattleStatus* battleStatus = &gBattleStatus;
+    s32 phi_s0;
+    s32 duration;
+
+    if (battleStatus->statusChance == 0xFE) {
```

Comment:
-2?

Reply:
Not equivalent, nor is changing the type of the field and using -2 or any permutation of those